src/Jackett.Common/Indexers/Xthor.cs

```diff
@@ -142,6 +147,106 @@ public class Xthor : BaseCachingWebIndexer
+                {"26", "Roumaines"},
+                {"27", "Malaisiennes"},
+                {"28", "Estoniennes"},
+                {"29", "Islandaises"},
+                {"30", "Grecques"},
+                {"31", "Serbes"},
+                {"32", "Norvégiennes"},
+                {"33", "Ukrainiennes"},
+                {"34", "Bulgares"},
+                {"35", "Tagalogues"},
+                {"36", "Xhosa"},
+                {"37", "Kurdes"},
+                {"38", "Bengali"},
+                {"39", "Amhariques"},
+                {"40", "Bosniaques"},
+                {"41", "Malayalam"},
+                {"42", "Télougou"},
+                {"43", "Bambara"},
+                {"44", "Catalanes"},
+                {"45", "Tchèques"},
+                {"46", "Afrikaans"}
+            })
+            { Value = "0" };
+            ConfigData.AddDynamic("specificLanguageAccent", ConfigSpecificLanguageAccent);
+
+            ConfigData.AddDynamic("advancedConfigurationWarning", new DisplayInfoConfigurationItem(string.Empty, "<center><b>Advanced Configuration</b></center>,<br /><br /> <center><b><u>WARNING !</u></b> <i>Be sure to read instructions before editing options bellow, you can <b>drastically reduce performance</b> of queries or have <b>non-accurate results</b>.</i></center><br/><br/><ul><li><b>Delay betwwen Requests</b>: (<i>not recommended</i>) you can increase delay to requests made to the tracker, but a minimum of 2.1s is enforced as there is an anti-spam protection.</li><br /><li><b>Max Pages</b>: (<i>not recommended</i>) you can increase max pages to follow when making a request. But be aware that others apps can consider this indexer not working if jackett take too many times to return results. Another thing is that API is very buggy on tracker side, most of time, results of next pages are same ... as the first page. Even if we deduplicate rows, you will loose performance for the same results. You can check logs to see if an higher pages following is not benefical, you will see an error percentage (duplicates) with recommandations.</li><br /><li><b>Bypass for TMDB</b>: (<i>recommended</i>) this indexer is compatible with TMDB queries (<i>for movies only</i>), so when requesting content with an TMDB ID, we will search directly ID on API instead of name. Results will be more accurate, so you can enable a max pages bypass for this query type. You will be at least limited by the hard limit of 4 pages.</li><br /><li><b>Enhanced Anime</b>: if you have \"Anime\", this will improve queries made to this tracker related to this type when making searches.</li><br /><li><b>Multi Replacement</b>: you can dynamically replace the word \"MULTI\" with another of your choice like \"MULTI.FRENCH\" for better analysis of 3rd party softwares.</li><li><b>Sub Replacement</b>: you can dynamically replace the word \"VOSTFR\" or \"SUBFRENCH\" with the word \"ENGLISH\" for better analysis of 3rd party softwares.</li></ul>"));
```

Comment:
`<center><b>Advanced Configuration</b></center>,<br /><br />` - the comma may be unnecessary, it's appearing as text

typo - `betwwen` > `between`

rephrase - `too many times` > `too long`

rephrase - `Another thing is that API is very buggy on tracker side, most of time, results of next pages are same ... as the first page.` > `Xthor's API is often very buggy, results on subsequent pages can be the same as the first.`

typo - `loose` > `lose`

rephrase - `if an higher pages following` > `if returning more pages`

typo - `recommandations` > `recommendations`

typo - `an TMDB ID` > `a TMDB ID`

rephrase - `we will search directly ID on API instead of name.` > `only the ID will be used, not the name.`

rephrase - `be at least limited` > `still be limited`

remove - Enhanced Anime, Multi Replacement, and Sub Replacement, it's just duplicating information.

Reply:
Thanks for suggestions, will apply them too